src/game/magic/RuneDraw.cpp

```diff
@@ -32,6 +32,8 @@ extern EERIE_CAMERA subj;
 
 static const Vec2s symbolVecScale(8*2, 6*2);
 
+SYMBOL_DRAW bookSymbolDraw;
```

Comment:
We use the g_ prefix for (new) global variables. You can change that in a follow-up patch though.